docker/docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
 
   instance10:
     image: postgres:10
+    command: ["-c", "shared_preload_libraries=pg_stat_statements"]
```

Comment:
This would override any 'shared_preload_libraries' setting in postgresql.conf (making shared_preload_libraries not configurable ultimately), right?

If so, a dedicated postgresql.conf file would be preferrable IMHO.

Reply:
It will be easier to use this form since it works for each version of PostgreSQL. If you use the postgresql.conf file, you would need a file per version.
Moreover, since this parameter is empty by default, I do not think this is a problem.
If you know of an other library to load, it might be easier to just include it in the docker-compose.xml file.

Reply:
Ok; fine by me.